### PR TITLE
feat: add doubleTap option for zoom

### PR DESCRIPTION
**Pull request by WoodNeck**
## Details
This adds `doubleTap` option for the option `zoom`

### CODE REVIEW

**Review by qpitlove — APPROVED** [review]
LGTM

File: src/control/ZoomControl.ts [hunk 1]
```diff
@@ -249,6 +266,7 @@ class ZoomControl extends Component<ControlEvents> implements CameraControl, Opt
     targetEl.addEventListener(BROWSER.EVENTS.WHEEL, this._onWheel, { passive: false, capture: false });
     targetEl.addEventListener(BROWSER.EVENTS.TOUCH_MOVE, this._onTouchMove, { passive: false, capture: false });
     targetEl.addEventListener(BROWSER.EVENTS.TOUCH_END, this._onTouchEnd, { passive: false, capture: false });
+    targetEl.addEventListener(BROWSER.EVENTS.DOUBLE_CLICK, this._onDoubleClick);
```
Comment (daybrush) [inline src/control/ZoomControl.ts:269]: Unlike other events, does a double click need no additional options?

**Review by daybrush — APPROVED** [review]
LGTM

File: src/control/ZoomControl.ts (hunk 1 above)
Comment (WoodNeck) [reply src/control/ZoomControl.ts:269]: `capture` needs to be false, so I guess it's fine without options.

**Review by malangfox — APPROVED** [review]
LGTM

**Pull request merged**